src/createAnimatedComponent.js

```diff
@@ -30,7 +30,7 @@ function hasAnimatedNodes(value) {
   if (Array.isArray(value)) {
     return value.some(item => hasAnimatedNodes(item));
   }
-  if (typeof value === 'object') {
+  if (typeof value === 'object' && value) {
```

Comment:
I'd suggested changing the order of them, following code uses established idiom for null checking and reads more naturally

```suggestion
  if (value && typeof value === 'object') {
```